src/style.css

```diff
@@ -1,3 +1,57 @@
+:root {
+  --link-color: #00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  font-family: Roboto, sans-serif;
```

Comment:
set it for html or body tag

Reply:
isn't fixed. please move this rule to selectors mentor asked you

Reply:
Still not fixed